rest_framework/schemas/generators.py

```diff
@@ -54,7 +54,7 @@ def endpoint_ordering(endpoint):
         'PATCH': 3,
         'DELETE': 4
     }.get(method, 5)
-    return (path, method_priority)
```

Comment:
Is there a reason this signature was changed?

Reply:
Yes, because otherwise the sort is by path and then method when we want to retain the order of paths and only sort on methods for a given path.